code/modules/research/machinery/departmental_circuit_imprinter.dm

```diff
@@ -3,10 +3,15 @@
 	desc = "A special circuit imprinter with a built in interface meant for departmental usage, with built in ExoSync receivers allowing it to print designs researched that match its ROM-encoded department type."
 	icon_state = "circuit_imprinter"
 	circuit = /obj/item/circuitboard/machine/circuit_imprinter/department
-	consoleless_interface = TRUE
 
 /obj/machinery/rnd/production/circuit_imprinter/department/science
 	name = "department circuit imprinter (Science)"
 	circuit = /obj/item/circuitboard/machine/circuit_imprinter/department/science
 	allowed_department_flags = DEPARTMENTAL_FLAG_ALL|DEPARTMENTAL_FLAG_SCIENCE
 	department_tag = "Science"
+
+/obj/machinery/rnd/production/circuit_imprinter/department/engineering
+	name = "department circuit imprinter (Science)"
+	circuit = /obj/item/circuitboard/machine/circuit_imprinter/department/science
```

Comment:
This should have a different circuit

Reply:
In fact, the name is also wrong? Not sure if it's actually used in-game, but it should be changed at least for the editor.

And are you sure about the department flags?